protos/Makefile

```
@@ -2,4 +2,9 @@
 go:
 	protoc --go_out=../go --go_opt=module=github.com/in-toto/attestation/go $(shell find ./in_toto_attestation -name "*.proto")
 
+# Creates the python libs in the python/ directory.
+python: 
+	mkdir -p ../python
+	protoc -I ./in_toto_attestation --python_out=../python $(shell find ./in_toto_attestation -name "*.proto")
```

I don't think we need the `-I` flag for this one. This will possibly also resolve not having the `in_toto_attestation` directory as part of the package. 

```suggestion
	protoc --python_out=../python $(shell find ./in_toto_attestation -name "*.proto")
```

---

@marcelamelara I actually included this to maintain consistency with the Go bindings. If we exclude this option then the bindings are placed in a child directory of `in_toto_attestation`.

e.g.

- python
  - in_toto_attestion
    - predicates
    - v1

---

@marcelamelara What is your preference?

---

@marcelamelara I see this comment in a later review 

`yes, I believe we need the in_toto_attestation directory as part of the python package directory hierarchy.` so I will operate under the assumption that you are expecting the directory to be present.

---

Yes, I think having that directory will make creating the python package a bit easier. Thanks again for making the changes!